pyproject.toml

```diff
@@ -16,6 +16,8 @@ duckdb = ">=0.6.0"
 sqlglot = ">=5.1.0"
 altair = ">=4.2.0"
 Jinja2 = ">=3.0.3"
+jellyfish = "^0.11.2"
+phonetics = "^1.0.5"
```

Comment:
They are used in the backend of `comparison_helpers.md` so I would assume they would have to be dependencies? Or if there is a different way of dealing with them, I am open to any ideas!

Reply:
I think in general we should try to keep the dependency list as slimmed down as possible.

`phonetics` doesn't seem to have any dependencies, so for now I think that's fine to add.
`jellyfish` - I would say let's remove that from the dependencies. The functions can be replaced w/ duckdb calls.

I would however suggest we keep the jellyfish functions. They can replace the duckdb dependency if it's not installed.

Reply:
Yeah - I second that - I think we def want to keep dependencies at absolute minimum.

If these are dependencies for the docs rather than for Splink itself, then we should manage the dependencies to build the docs separately

Reply:
Thanks both for your input. I agree that For clarity, these functions are more to help users get an intuition for comparators, not just for docs (apologies, I realise I had referred to this as `comparison_helpers.md` above, but that was in error.